flake.nix

```diff
@@ -0,0 +1,66 @@
+{
+  description = ''
+    Set of patches to the standard unsquashfs utility (part of
+    squashfs-tools) that attempts to add support for as many hacked-up
+    vendor-specific SquashFS implementations as possible
+  '';
+
+  inputs.nixpkgs.url = "nixpkgs/nixpkgs-unstable";
+
+  outputs = { self, nixpkgs }:
+    let
+      # Generate a user-friendly version number.
+      version = builtins.substring 0 8 self.lastModifiedDate;
+
+      # System types to support.
+      supportedSystems = [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" "aarch64-darwin" ];
```

Comment:
I would remove mentions of Darwin in this branch since the CI only builds for Linux.